### PR TITLE
CATROID-1042 Crash ArrayIndexOutOfBoundsException in StageListener.java:282

### PR DESCRIPTION
**Pull request by CodeChamp-SS**
Fixed ArrayIndexOutOfBoundsException in StageListener.java:282.

https://jira.catrob.at/browse/CATROID-1042

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [ ] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [ ] Include reasonable and readable tests verifying the added or changed behavior
- [ ] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [ ] After the PR, verify that all CI checks have passed
- [ ] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by mstoeg — CHANGES_REQUESTED** [review]
Unfortunately, I can't approve this PR unless there is a way to reproduce this bug or at least a testcase that verifies your bugfix. Moreover what if the issue in the first place was that somehow `cloneMe.look` is null and therefore not contained in the ActorList? Then you would add a null element to the list.

**Comment by CodeChamp-SS** [comment]
> Unfortunately, I can't approve this PR unless there is a way to reproduce this bug or at least a testcase that verifies your bugfix. Moreover what if the issue in the first place was that somehow `cloneMe.look` is null and therefore not contained in the ActorList? Then you would add a null element to the list.

hmm, but I think if `cloneMe.look` was a null element then it would have given a nullptr exception too

**Comment by CodeChamp-SS** [comment]
I understand that it's important to know how to reproduce the bug too, to verify the changes : )

**Pull request closed without merging**
